### PR TITLE
Certbot auto timeout

### PR DESCRIPTION
**Pull request by erikrose**
A continuation of #4929

### CODE REVIEW

**Comment by erikrose** [comment]
We'll need 2 review for this, since it's certbot-auto, of course.

File: letsencrypt-auto-source/pieces/fetch.py [hunk 1]
```diff
@@ -56,7 +57,9 @@ def get(self, url):
 
         """
         try:
-            return self._opener.open(url).read()
+            // socket module docs say default timeout is None: that is, no
```
Comment (bmw) [inline letsencrypt-auto-source/pieces/fetch.py]: *gasps*

Erik's been cheating on Python! That doesn't look like Pythonic comment syntax to me.

Comment (erikrose) [reply letsencrypt-auto-source/pieces/fetch.py]: How embarrassing! I thought it didn't look right, but I couldn't put my finger on it!

**Comment by erikrose** [comment]
Of course, rebasing will dissociate this PR from the earlier one, but if it makes you happy, I'm happy to make you happy.

**Pull request merged**